packages/dbos-cloud/databases/databases.ts

```diff
@@ -333,6 +342,10 @@ export async function connect(host: string, dbName: string, password: string) {
       return 1;
     }
 
+    const backupConfigFilePath = `dbos-config.yaml.${generateRandomSuffix(6)}.bak`;
```

Comment:
Is a random suffix better than a date stamp?

Reply:
Good point, fixed